otx/algorithms/anomaly/tasks/inference.py

```diff
@@ -337,8 +341,7 @@ def _is_docker() -> bool:
 
     def unload(self) -> None:
         """Unload the task."""
-        if os.path.exists(self.config.project.path):
-            shutil.rmtree(self.config.project.path, ignore_errors=False)
+        self.cleanup()
 
         if self._is_docker():
```

Comment:
Wouldn't it still give a segmentation fault on this code? It looks like we need to check in CI.

Reply:
This confirmed that the existing issue was not reproducible in CI.